tests/destructor/tnewruntime_strutils.nim

```diff
@@ -214,5 +212,3 @@ staticTests()
 let xaa = @[""].join()
 let xbb = @["", ""].join()
 
-let (a, d) = allocCounters()
```

Comment:
This needs to check for ``getOccupiedMem() == 0`` instead then so that we have some leak detection for the targets that happen to not use ``valgrind``. Hmm but then the ``-d:useMalloc`` does bite. So we need a testament patch, ``valgrind: true`` should imply ``-d:useMalloc``.